compiler/il/Aliases.cpp

```diff
@@ -362,7 +362,9 @@ OMR::SymbolReference::getUseDefAliasesBV(bool isDirectCall, bool includeGCSafePo
             }
 
          if (symRefTab->isNonHelper(self(), TR::SymbolReferenceTable::arrayCmpSymbol) ||
-             symRefTab->isNonHelper(self(), TR::SymbolReferenceTable::arrayCmpLenSymbol))
+             symRefTab->isNonHelper(self(), TR::SymbolReferenceTable::arrayCmpLenSymbol) ||
+	     symRefTab->isNonHelper(self(), TR::SymbolReferenceTable::jProfileValueSymbol) ||
```

Comment:
This line starts with a tab while the vast majority of lines in JIT source don't use tabs at all.

Reply:
Thanks, I'll fix shortly.